scripts/ts-bin/pair_comments_for_relativerating.ts

```diff
@@ -19,158 +19,170 @@ Usage: Convert answers with multiple parts into separate rows.
   ts-node ./ts-bin/pair_comments_for_relativerating.ts \
     --infile="./tmp/path-to-input.jsonl" \
     --output_path_prefix="./tmp/path-to-paired"
+
+  Once this has been done, the jsonlines can be converted to CSV file using:
+  ./node_modules/.bin/json2csv --ndjson \
+    --input="./tmp/path-to-paired.jsonl" \
+    --output="./tmp/path-to-paired.csv"
 */
 
-import * as yargs from 'yargs';
 import * as fs from 'fs';
 import * as readline from 'readline';
-// import * as jsonexport from "jsonexport/dist"
-// import * as stream from 'stream';
+import * as yargs from 'yargs';
 
 // Command line arguments.
 interface Params {
-  infile:string,
-  output_path_prefix:string,
-  output_path_postfix:string,
-  left_id_prefix:string,
-  right_id_prefix:string,
-  left_text_prefix:string,
-  right_text_prefix:string,
-  entries_per_row:number,
-  rows_per_file:number,
-};
+  infile: string, output_path_prefix: string, output_path_postfix: string,
```

Comment:
Ugh...I hate what the autoformatter did here, but I guess I did ask you to run it! =) I'll make a mental note to ask about the reasoning of some of these format options internally.